flink-dist/src/main/flink-bin/bin/config.sh

```diff
@@ -603,38 +623,32 @@ TMSlaves() {
 
 runBashJavaUtilsCmd() {
     local cmd=$1
-    local class_path=$2
-    local conf_dir=$3
-    local EXECUTION_PREFIX="BASH_JAVA_UTILS_EXEC_RESULT:"
+    local conf_dir=$2
+    local class_path="${3:-$FLINK_BIN_DIR/bash-java-utils.jar:`findFlinkDistJar`}"
```

Comment:
`findFlinkDistJar` will already add `:`

Reply:
According to the other comment I reworked the findFlinkDistJar method. It does not longer return the `:`.